src/main/java/ua/kpi/iasa/IASA_Organiser/data/impl/EventStore.java

```diff
@@ -1,7 +1,7 @@
 package ua.kpi.iasa.IASA_Organiser.data.impl;
 
 import ua.kpi.iasa.IASA_Organiser.model.Event;
-
+@Deprecated
 public class EventStore {  //smth like database/it`s foundation
```

Comment:
Remove redundant comment;
(Never use write comments with '//', write TODO or better java docs)

Reply:
Fix it pls